.env

```diff
@@ -1 +1 @@
-REACT_APP_JWT_SECRET=deekshithmogra
\ No newline at end of file
+REACT_APP_JWT_SECRET=deekshithmograecommerce
```

Comment:
Remove the env file, as it should not be uploaded on Github. You can refer to the below article for help https://daily-dev-tips.com/posts/removing-a-env-file-from-git-history/

Reply:
ok, I will check that.